eZ/Publish/API/Repository/Values/Content/Query/Criterion/Sibling.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+
+use eZ\Publish\API\Repository\Values\Content\Location;
+
+/**
+ * A criterion that matches content that is sibling to the given Location.
+ */
+class Sibling extends CompositeCriterion
+{
+    public function __construct(int $id, int $parentLocationId)
```

Comment:
DX, how about:
```suggestion
    public function __construct(int $locationId, int $parentLocationId)
```

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/2958/commits/623be7ef86c0cc9e8840c8f59a574095a8c33cac